lib/sedna/algorithms/knowledge_management/task_evaluation/task_evaluation.py

```diff
@@ -0,0 +1,80 @@
+from sedna.common.config import Context
+from sedna.common.log import LOGGER
+from sedna.common.class_factory import ClassType, ClassFactory
+
```

Comment:
add comment to explain why task evaluation is needed

Reply:
done